Source/mpq/mpq_writer.hpp

```diff
@@ -56,11 +64,11 @@ class MpqWriter {
 
 	LoggedFStream stream_;
 	std::string name_;
-	std::uintmax_t size_;
-	bool modified_;
-	bool exists_;
-	MpqHashEntry *hashTable_;
-	MpqBlockEntry *blockTable_;
+	std::uintmax_t size_ {};
+	bool modified_ {};
+	bool exists_ {};

```

Comment:
There's no point in having this as a field. It should be local to the constructor.